### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/components.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4867]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4867|
|Type|enhancement|
|Created|2011-02-11 05:23:40Z|
|Branch|https://github.com/twisted/twisted/tree/components-sphinx-4867|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the components document

<details><summary>Searchable metadata</summary>

```
trac-id__4867 4867
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_components_sphinx_4867 branches/components-sphinx-4867
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297401820000000 1297401820000000
changetime__1300137565000000 1300137565000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30666]) Branching to 'components-sphinx-4867'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30667]) Merge the components parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4867](https://github.com/twisted/twisted/issues/4867)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_6"></a>|@tenth10th set owner to Screwtape|
|-|-|

There are a lot of changes here due to the change from 110V to 120V and "Foreign" to "UK", but everything seems to be in order.

The only inconsistency I noticed was that the doc string on the IAmericanSocket Interface's voltage() method is on the same line as the first set of triple quotes, unlike the other comments (which have been modified to have the triple quotes and text on separate lines):
```
class IAmericanSocket(Interface):
    def voltage():
      """Return the voltage produced by this socket object, as an integer.
      """
```

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn set owner to @lvh|
|-|-|

Review feedback isn't really part of the conversion process, if it's still an issue, please create a new ticket.

Other than that, this one can be merged.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_9"></a>|@lvh set status to `closed`|
|-|-|

(In [31059]) Better lore2sphinx output for components.xhtml

Author: khorn, Screwtape, jdb
Reviewer: tenth, lvh
Fixes: [#4867](https://github.com/twisted/twisted/issues/4867)

"Make Sphinx look gooder" -- JP
